js/src/pages/settings/enhanced-conversions/setup-enhanced-conversions.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import { CheckboxControl } from '@wordpress/components';
+import { useCallback, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppDocumentationLink from '~/components/app-documentation-link';
+import { useEnableEnhancedConversions } from './useEnableEnhancedConversions';
+import Section from '~/components/section';
+import useDispatchCoreNotices from '~/hooks/useDispatchCoreNotices';
+import { useAppDispatch } from '~/data';
+
+const SetupEnhancedConversions = () => {
+	const isEnabled = useEnableEnhancedConversions();
+	const [ isSaving, setIsSaving ] = useState( false );
+	const { createNotice } = useDispatchCoreNotices();
+	const { updateEnhancedConversionsStatus } = useAppDispatch();
+
+	const toggleEnhancedConversions = useCallback( async () => {
+		await updateEnhancedConversionsStatus( ! isEnabled );
+	}, [ updateEnhancedConversionsStatus, isEnabled ] );
+
+	const handleOnChange = async () => {
+		try {
+			setIsSaving( true );
+			await toggleEnhancedConversions();
+
+			createNotice(
+				'success',
+				__(
+					'Enhanced Conversions status updated successfully.',
+					'google-listings-and-ads'
+				)
+			);
+		} catch ( error ) {
+			// Silently fail because the error is handled in the hook.
```

Comment:
@ankitrox Can you please update the comment since we are no longer using the hook now.